Dockerfile

```diff
@@ -29,6 +29,7 @@ COPY packages/cardano-services-client/package.json packages/cardano-services-cli
 COPY packages/core/package.json packages/core/package.json
 COPY packages/crypto/package.json packages/crypto/package.json
 COPY packages/dapp-connector/package.json packages/dapp-connector/package.json
+COPY packages/dapp-connector-client/package.json packages/dapp-connector-client/package.json
```

Comment:
Dockerfile is for building the backend, it shouldn't need this package
```suggestion
```

Reply:
Without it it breaks, and I can see the `dapp-connector` it's added. I assumed there must have been a reason for that?

Reply:
I see it adds all packages, nevermind let's keep it like that, although seems like a hack